build/bakefiles/mac_bundles.bkl

```diff
@@ -64,6 +64,9 @@
 
                 <!-- copy the application icon: -->
                 cp -f $(BUNDLE_ICON) $(BUNDLE)/Resources/wxmac.icns
+
+                cp -f $(WXTOPDIR)tests/horse.png $(BUNDLE)/Resources/horse.png
+                cp -f $(WXTOPDIR)tests/horse.cur $(BUNDLE)/Resources/horse.cur
```

Comment:
I don't think this is specific to tests? Is this also copied into the bundle of all the samples? Based on all the modified `Makefile.in` it seems it is.
And CMake might need something similar.

Reply:
@MaartenBent ,
I have no idea how to fix CMake thing.
But it seems it doesn't fail...

Reply:
The last CMake (and other) CI runs were 7 months ago. Maybe you can fix the merge conflicts so the CI tests can run.